### PR TITLE
TASK-51421: Bookmark/unbookmark a news

### PR DESCRIPTION
**Pull request by aelbenney**

### CODE REVIEW

**Review by nidhalnidhal — APPROVED** [review]

**Review by azayati — CHANGES_REQUESTED** [review]
We have a big refactoring in progress, please don't merge anything on develop before merging the refactoring

File: services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java
Comment (azayati) [inline services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java:922]: We should not call directly Metadata service from rest layer neither from service layer.
Rest layer should call service layer and service layer should call favorite service

**Review by azayati — CHANGES_REQUESTED** [review]
See review

**Pull request merged**
